src/function/scalar/string/damerau_levenshtein.cpp

```diff
@@ -0,0 +1,106 @@
+#include "duckdb/function/scalar/string_functions.hpp"
+
+#include <map>
```

Comment:
We have wrapped include+using of std containers in duckdb, if you change `#include <map>` with `#include "duckdb/common/map.hpp"` that should allow you to just use unqualified `map` in the codebase.

The same goes for `std::vector`, here it's actually more important to use `#include "duckdb/common/vector.hpp"` because we have our own `duckdb::vector` implementation, that provides a little more memory safety.
In this case you're not returning/receiving vectors from other internal methods, so it's not crucial to use it, but I'd recommend to do so anyways

Reply:
Ah okay, apologies I had not realised that. Will do 👍